01week/datatypes.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+//using console logs to test
+
+//simple date and time
+const day = new Date();
+console.log(day);
+
+//turning number into string using '.toString'
+const numToStr = 1337;
+const nTS = numToStr.toString();
+console.log(nTS);
+
+//turning string into number using 'parseInt'
+const strToNum = parseInt("112358");
+console.log(strToNum);
+
+//typeof commands inside console logs
+console.log(typeof true);
+console.log(typeof null); //null is an Object for some reason//
+console.log(typeof undefined);
+console.log(typeof NaN);
+console.log(typeof "Words");
+
+//simple math
+const addi = 4+6;
```

Comment:
This method is fine, however the instructions were to 'write a program' which means 'write a function'. Please wrap in a function.